.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 **/*.rs.bk
+src/asset.in
+frontend/dist/
```

Comment:
Yes that would be the case unfortunately. However, I'm not really sure how this would be done without that. Dynamic means we need whatever creates it unfortunately. Otherwise we'd have to still update the assets manually. Which is possible but kind of a pain. :/ I'm not really sure what the best course of action would be.